frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1027,14 +1017,19 @@
             vm.remainingTime = {};
             vm.showClock = false;
             vm.showSubmissionNumbers = false;
+            vm.maxExceeded = false;
             parameters.url = "jobs/" + vm.challengeId + "/phases/" + phaseId + "/remaining_submissions";
             parameters.method = 'GET';
             parameters.callback = {
                 onSuccess: function(response) {
                     var status = response.status;
                     var details = response.data;
                     if (status === 200) {
-                        if (details.remaining_submissions_today_count > 0) {
```

Comment:
This line actually came from master branch... I just pasted the code from master branch of `frontend/src/js/controllers/challengeCtrl.js` and then did my updates.